File: javastory.budgetsh.stage2/src/javastory/budgetsh/stage2/budget/storage/store/CashBookStore.java
package javastory.budgetsh.stage2.budget.storage.store;

import javastory.budgetsh.stage2.budget.entity.budget.CashBook;

public interface CashBookStore {
	//
	public boolean exist(String bankAccount);
	public CashBook regist(CashBook cashbook);
	public CashBook retrieve(String account);
	public void update(CashBook foundCashbook);
	public void remove(CashBook foundCashbook);

}

File: javastory.budgetsh.stage5.1/src/step3/server/store/filestore/CashbookStore.js
const CashbookFile = require('./io/CashbookFile.js');

exports.exist = function(bankAccount) {
	if (CashbookFile.exist(bankAccount)) {
		return true;
	} else {
		return false;
	}
}

exports.retrieve = function(bankAccount) {
	return CashbookFile.retrieve(bankAccount);
}

exports.update = function(cashbook) {
	CashbookFile.update(cashbook);
}

exports.remove = function(cashbook) {
	CashbookFile.remove(cashbook);
}

exports.regist = function(cashbook) {
	CashbookFile.regist(cashbook);
}